weather.py

```diff
@@ -0,0 +1,198 @@
+import argparse
+import os
+import calendar
+import csv
+
+required_labels = [
+    "PKT", "PKST", "Max TemperatureC", "Min TemperatureC", "Mean Humidity"
+]
+
+month_abbr_map = {index: month for index, month in enumerate(calendar.month_abbr) if month}

```

Comment:
Instead of creating map can't we directly consume calendar utilities for getting month name?